Celeste.Mod.mm/Patches/Level.cs

```diff
@@ -53,8 +55,28 @@ class patch_Level : Level {
         public extern void orig_RegisterAreaComplete();
         public new void RegisterAreaComplete() {
             bool completed = Completed;
-            orig_RegisterAreaComplete();
-            if (!completed) {
+            //orig_RegisterAreaComplete();
```

Comment:
Might as well just remove this comment, the orig_ definition and mark this method with MonoModReplace.

Reply:
-pluck-